std/traits.d

```diff
@@ -2435,17 +2435,30 @@ if (is(T == function))
     alias SetFunctionAttributes = FunctionTypeOf!(SetFunctionAttributes!(T*, linkage, attrs));
 }
 
-///
+/// Use with extreme caution as this can easily result in Undefined Behavior
 @safe unittest
 {
     alias ExternC(T) = SetFunctionAttributes!(T, "C", functionAttributes!T);
 
-    auto assumePure(T)(T t)
+    auto assumePure(T)(T t) @trusted
```

Comment:
This is reinforcing the dangerous idea of exposing unsafe functions as trusted. Shouldn't the example have `g()` trusted instead? Then we can remove the doc-comment warning.